x/shield/types/types.go

```diff
@@ -6,6 +6,23 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// Global variables
+// 
+// 	// TotalCollateral is the amount of all collaterals in the pool.
+// 	TotalCollateral sdk.Int `json:"total_collateral" yaml:"total_collateral"`
+//
+//	// TotalWithdrawing is the amount of collateral in withdraw queues.
+//	TotalWithdrawing sdk.Int `json:"withdrawing" yaml:"withdrawing"`
```

Comment:
@dshe-certik I don't see the need for `TotalWithdrawing` since we have `provider.Withdrawing`. I think we were using `collateral.Withdrawing` incorrectly. Am I missing something?

Reply:
Right I don't think we need it for now.